appinventor/misc/mavendl/mavendl.py

```diff
@@ -0,0 +1,128 @@
+# Create mavendl to download AAR libs with transitive deps 
+# This stores the files in a "libs" directory relative to this script
+# Usage: mavendl.py <group>:<artifact>:<version>
+
+import argparse
+import os
+import urllib.request
+from io import BytesIO
+from typing import Dict, List, Set, Tuple
+from xml.dom import minidom
+from zipfile import ZipFile
+
+parser = argparse.ArgumentParser()
+parser.add_argument("coord")
```

Comment:
Consider add the `help=` keyword to be clear all three parts of the coordinate need to be specified.